docker-for-windows/faqs.md

```diff
@@ -111,62 +95,55 @@ learn more about running diagnostics and about Docker Desktop for Windows GitHub
-Inc. will delete uploaded diagnostics bundles after 30 days unless they are
-referenced in an open issue on the
+who are directly involved in diagnosing Docker Desktop issues. By default, Docker Inc. will delete uploaded diagnostics bundles after 30 days unless they are referenced in an open issue on the
 [docker/for-mac](https://github.com/docker/for-mac/issues) or
 [docker/for-win](https://github.com/docker/for-win/issues) issue trackers. If an
 issue is closed, Docker Inc. will remove the referenced diagnostics bundles
 within 30 days. You may also request the removal of a diagnostics bundle by
-either specifying the diagnostics ID or via your GitHub ID (if the diagnostics
-ID is mentioned in a GitHub issue). Docker Inc. will only use the data in the
-diagnostics bundle to investigate specific user issues, but may derive high
-level (non personal) metrics such as the rate of issues from it.
+either specifying the diagnostics ID or through your GitHub ID (if the diagnostics ID is mentioned in a GitHub issue). Docker Inc. will only use the data in the diagnostics bundle to investigate specific user issues, but may derive high level (non-personal) metrics such as the rate of issues from it.
 
-### Can I use Docker Desktop for Windows with new swarm mode?
+### Can I use Docker Desktop with the Swarm mode?
 
-Yes! You can use Docker Desktop for Windows to test single-node features of
+Yes! You can use Docker Desktop to test single-node features of
 [swarm mode](/engine/swarm/) introduced with Docker Engine 1.12, including
 initializing a swarm with a single node, creating services, and scaling
 services. Docker “Moby” on Hyper-V serves as the single swarm node. You can also
-use Docker Machine, which comes with Docker Desktop for Windows, to create and
+use Docker Machine, which is bundled in Docker Desktop, to create and
 experiment with a multi-node swarm. Check out the tutorial at
 [Get started with swarm mode](/engine/swarm/swarm-tutorial/).
 
 ### How do I connect to the remote Docker Engine API?
 
 You might need to provide the location of the Engine API for Docker clients and development tools.
 
-On Docker Desktop for Windows, clients can connect to the Docker Engine through a
+On Docker Desktop, clients can connect to the Docker Engine through a
 **named pipe**: `npipe:////./pipe/docker_engine`, or **TCP socket** at this URL:
 `tcp://localhost:2375`.
 
 This sets `DOCKER_HOST` and `DOCKER_CERT_PATH` environment variables to the
 given values (for the named pipe or TCP socket, whichever you use).
 
-See also [Docker Engine API](/engine/api) and the Docker Desktop for Windows forums
-topic
-[How to find the remote API](https://forums.docker.com/t/how-to-find-the-remote-api/20988){: target="_blank" class="_"}.
+See also [Docker Engine API](/engine/api) and the Docker Desktop for Windows forums topic [How to find the remote API](https://forums.docker.com/t/how-to-find-the-remote-api/20988){: target="_blank" class="_"}.
```

Comment:
I noticed the change from "Docker Desktop for Windows" to just "Docker Desktop" in other sections of this doc. Does that apply here as well?

Reply:
Left this instance 'as is' because the forum is still called `Docker Desktop for Windows`  - https://forums.docker.com/c/docker-desktop-for-windows